Modules/Core/Sources/Core/Buffer/Buffer.swift

```diff
@@ -6,7 +6,136 @@ import Darwin
 #endif
 
 public typealias Byte = UInt8
-public typealias Buffer = DispatchData
+
+public struct Buffer : RandomAccessCollection {
+    public typealias Iterator = BufferIterator
+    public typealias Index = Int
+    public typealias Indices = DefaultRandomAccessIndices<Buffer>
+    
+    public private(set) var bytes: ContiguousArray<Byte>
+    
+    public var count: Int {
+        return bytes.count
+    }
+    
+    public init(_ bytes: [Byte] = []) {
+        self.bytes = ContiguousArray(bytes)
+    }
+    
+    public init(bytes: [Byte]) {
+        self.bytes = ContiguousArray(bytes)
+    }
+    
```

Comment:
Is this initializer is required by `RandomAccessCollection`? no, right? why do we need this if we have the same initializer without labels above?